Consider all statuses that represent Done

codificat opened the pull request:
An issue which is marked as Obsolete or Won't Fix is also "Done" as there is no more work required for it. Therefore, the list of completed issues should also include Obsolete or Won't Fix issues.

For reference see Stu/Pep's comments in KNIECO-1648

dankenigsberg commented on jira/issue.go:
```diff
@@ -73,22 +73,31 @@ const (
 	IssueStatusCodeReview IssueStatus = "Code Review"
 
-	// IssueStatusQEReview represents the Issue Status Done
+	// IssueStatusQEReview represents the Issue Status QE Review
```
heh, you were annoyed by these, too. My opinion is to remove these self-evident comments.

codificat replied:
:) yeah, you're right, it's better to just remove them

dankenigsberg commented on jira/issue.go:
```diff
@@ -146,6 +155,20 @@ func (i *Issue) IsActive() bool {
 }
 
+// IsDone returns true if no more work is needed in the issue
+func (i *Issue) IsDone() bool {
```
I think that this should be named IsResolved; it may even refer to the Resolution field instead of the Status one.

codificat replied:
Actually, `IsResolved` [already exists](https://github.com/dankenigsberg/jiracsv/blob/d87abb63eb8e1e77cb9f11de9b3825c50281f6b5/jira/issue.go#L163-L170) (and it uses the resolution field as you say)... but it was not clear to me if this would catch Obsolete/Won't Fix issues properly, hence why I added this one based on status instead of relying on `IsResolved`, just to be sure. This was due to my lack of Jira knowledge, though ;)

If `Resolution` is awlays set to `Done` in all types of issue closures (Done, Obsolete, Won't Fix) then yes it's better to just use that existing function.

codificat committed:
Remove unnecessary comments

Some comments about issue status / type / priority were redundant or even not up
to date.

codificat committed:
Add additional statuses

Our boards have issues with statuses that were not in the list of possible
statuses.

codificat committed:
Consider all resolved issues as Done

There are multiple ways to resolve an issue, which translate to different
statuses, including Done but also Obsolete or even "Won't Fix /
Obsolete". Instead of looking at just one possible status (Done), check the
resolution to determine if an issue is Done.

codificat commented:
I beliete this project is not in use anymore, so I'm going to close this one.
